app/components/article-media-thumbnail.js

```diff
@@ -14,8 +16,6 @@ export default Ember.Component.extend(
 		classNameBindings: ['itemType', 'isLoading', 'isSmall'],
 		tagName: 'figure',
 
-		layoutName: 'components/article-media-thumbnail',
```

Comment:
Are you 100% sure it's not needed anymore? IIRC there were issues in galleries without it.

Reply:
It's injected in render-component,